### PR TITLE
OIDCNG well known configuration

### PR DESCRIPTION
**Pull request by oharsta**

### CODE REVIEW

File: roles/oidcng/templates/openid-configuration.json.j2 [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "issuer": "https://org.openconext.oidc.ng",
```
Comment (quartje) [inline roles/oidcng/templates/openid-configuration.json.j2]: Shouldn't this point to https://oidcng.{{ base_domain }} ?

File: roles/oidcng/templates/openid-configuration.json.j2 [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    "uids"
+  ],
+  "claims_parameter_supported": true,
+  "acr_values_supported": [
+    "http://{{ env }}.surfconext.nl/assurance/loa1",
+    "http://{{ env }}.surfconext.nl/assurance/loa2",
+    "http://{{ env }}.surfconext.nl/assurance/loa3",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level2",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level3"
+  ],
```
Comment (quartje) [inline roles/oidcng/templates/openid-configuration.json.j2]: too much SURFconextisms here. Can we make this switchable, and the values configurable?

File: roles/oidcng/tasks/main.yml [hunk 3]
```diff
@@ -63,6 +63,13 @@
   notify:
     - "restart oidcng"
 
+- name: copy oidcng well-known discovery configuration
+  template:
+    src: "openid-configuration.json.j2"
+    dest: "{{ oidcng_dir }}/openid-configuration.json"
+  notify:
+    - "restart oidcng"
```
Comment (quartje) [inline roles/oidcng/tasks/main.yml]: Do we need to restart or would a reload be sufficient?

Comment (oharsta) [reply roles/oidcng/tasks/main.yml]: Restart and reload are the same for a Spring boot application

File: roles/oidcng/templates/openid-configuration.json.j2 (hunk 2 above)
Comment (oharsta) [reply roles/oidcng/templates/openid-configuration.json.j2]: https://github.com/OpenConext/OpenConext-deploy/commit/93e49d1106123949a163a59de7b4f47d9a61c031

File: roles/oidcng/templates/openid-configuration.json.j2 (hunk 1 above)
Comment (oharsta) [reply roles/oidcng/templates/openid-configuration.json.j2]: Yep. Changed this.

**Pull request merged**